Codecademy/gamut
# feat: Rename accordion to deprecated

### Overview


Renamed `Accordion`, `AccordionArea`, and `AccordionButton` to `AccordionDeprecated`, `AccordionAreaDeprecated`, and `AccordionButtonDeprecated` to support our NEW accordion that will be coming in a follow up PR.


### PR Checklist

- [x] Related to JIRA ticket: [WEB-2437]

#### Testing Instructions



Visit https://64e77a589bd1711f953f88c8--gamut-preview.netlify.app/ and confirm accordion story is renamed as deprecated

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/36383)  | [Monolith Env](https://pr-36383-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/3321)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-3321)   |




[WEB-1767]: https://codecademy.atlassian.net/browse/WEB-1767?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[WEB-2437]: https://codecademy.atlassian.net/browse/WEB-2437?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
